nc_news frontend/src/util/api.jsx

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+const ncMarketPlace = axios.create({
+  baseURL: "https://clsc-nc-news.onrender.com/api",
+});
+
+export const fetchArticles = (category = "") => {
+  return ncMarketPlace.get(`/articles`);
+};
```

Comment:
Nice work with axios 👍  

You could narrow down what you return here so you don't have to destructure inside the useEffect. If you chain a .then you can return res.data.articles here